autogpt_platform/frontend/package.json

```diff
@@ -8,7 +8,10 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "test": "playwright test",
+    "test-ui": "playwright test --ui",
```

Comment:
what's the difference between these two?

any chance you could add a bit of info about this playwright setup in the (advanced) README?

Reply:
when its ready yeah, gotta get other stuff workin first